### PR TITLE
feat: upgrade sentencizer to 3.0

### PR DESCRIPTION
**Pull request by cristianmtr**
Goals:

- upgrade Sentencizer to 3.0
- ...

- [x] Bumped version (in README of Executor)

PR feedback: @jina-ai/workstream-hub

### CODE REVIEW

File: jinahub/segmenters/Sentencizer/sentencizer.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def __init__(
         max_sent_len: int = 512,
         punct_chars: Optional[List[str]] = None,
         uniform_weight: bool = True,
-        traversal_paths: Tuple[str] = ('r',),
+        traversal_paths: str = 'r',
```
Comment (azayz) [inline jinahub/segmenters/Sentencizer/sentencizer.py:26]:
```suggestion
        traversal_paths: str = '@r',
```

File: jinahub/segmenters/Sentencizer/sentencizer.py [hunk 2]
```diff
@@ -37,7 +37,7 @@ def __init__(
             for example ['!', '.', '?'] will use '!', '.' and '?'
         :param uniform_weight: the definition of it should have
             uniform weight or should be calculated
-        :param traversal_paths: traverse path on docs, e.g. ['r'], ['c']
+        :param traversal_paths: traverse path on docs, e.g. 'r', 'r,c'
```
Comment (azayz) [inline jinahub/segmenters/Sentencizer/sentencizer.py:40]:
```suggestion
        :param traversal_paths: traverse path on docs, e.g. '@r', '@r,c'
```

File: jinahub/segmenters/Sentencizer/tests/unit/test_exec.py [hunk 3]
```diff
@@ -10,7 +10,7 @@ def test_config():
     assert ex.min_sent_len == 1
 
 
-@pytest.mark.parametrize('traversal_paths', [('r',), ('c',)])
+@pytest.mark.parametrize('traversal_paths', ['r', 'c'])
```
Comment (azayz) [inline jinahub/segmenters/Sentencizer/tests/unit/test_exec.py:13]:
```suggestion
@pytest.mark.parametrize('traversal_paths', ['@r', '@c'])
```

**Review by azayz — CHANGES_REQUESTED** [review]

File: jinahub/segmenters/Sentencizer/tests/integration/test_exec.py [hunk 4]
```diff
@@ -10,7 +10,7 @@ def test_exec():
             inputs=Document(text='Hello. World! Go? Back'),
             return_results=True,
```
Comment (JoanFM) [inline jinahub/segmenters/Sentencizer/tests/integration/test_exec.py:11]:
```suggestion
```

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Comment by cristianmtr** [comment]
Not sure why the integration test is hanging. It's passing locally very fast. @JoanFM @azayz can you approve?

**Comment by JoanFM** [comment]
Why are there multiple invocations of integration tests?

**Review by JoanFM — CHANGES_REQUESTED** [review]
@cristianmtr let's use this PR to move it to a different repo. I think this may be the root problem

**Comment by cristianmtr** [comment]
> Why are there multiple invocations of integration tests?

I only see one. 
image

**Comment by cristianmtr** [comment]
> @cristianmtr let's use this PR to move it to a different repo. I think this may be the root problem

It's been split here , but Sentencizer is not used in any of them
image

**Review by azayz — APPROVED** [review]

**Pull request merged**